### PR TITLE
ENG-1917 - feat: turns `tokenize` into a SDK method

### PR DESCRIPTION
**Pull request by djejaquino**
⚠️  Coordinate release with elements, hosted elements and docs  ⚠️ 

## Description

Turns `BasisTheory#tokenize` into a method the user may invoke directly

## Testing required outside of automated testing?

- [x] Not Applicable

### Screenshots (if appropriate):

- [x] Not Applicable

## Rollback / Rollforward Procedure

- [x] Roll Forward
- [ ] Roll Back

## Reviewer Checklist

- [ ] Description of Change
- [ ] Description of outside testing if applicable.
- [ ] Description of Roll Forward / Backward Procedure
- [ ] Documentation updated for Change

### CODE REVIEW

**Review by brigonzalez — DISMISSED** [review]

File: library/test/setup/utils.ts [hunk 1]
```diff
@@ -547,6 +547,132 @@ const testCRUD = <T, C, U>(
...
+      });
+
+      test('should delegate create call to bt.js', () => {
+        const superMethod = jest.fn().mockReturnValue(expectedCreatedToken);
+        const ClassToBeInstantiated = delegateServiceUnderTest(
+          (elementsInstance as unknown) as BasisTheoryElementsInternal
+        );
+
+        Object.setPrototypeOf(ClassToBeInstantiated.prototype, {
+          create: superMethod,
```
Comment (brigonzalez) [inline library/test/setup/utils.ts:617]: 💣

File: library/test/setup/utils.ts [hunk 2]
```diff
@@ -547,6 +547,132 @@ const testCRUD = <T, C, U>(
...
+        const serviceInstance = new ClassToBeInstantiated({
+          apiKey: chance.string(),
+          baseURL: chance.url(),
+        });
+        const expectedPayload = chance.string(),
+          expectedRequestOptions = chance.string();
+
+        const createdToken = ((serviceInstance as unknown) as {
+          tokenize?: (...args: unknown[]) => string;
+          create?: (...args: unknown[]) => string;
```
Comment (brigonzalez) [inline library/test/setup/utils.ts:629]: 💣

File: library/test/setup/utils.ts [hunk 3]
```diff
@@ -547,6 +547,132 @@ const testCRUD = <T, C, U>(
...
+    beforeEach(() => {
+      (hasElement as jest.Mock).mockReturnValue(false);
+    });
+
+    test('should delegate create call to bt.js', () => {
+      const superMethod = jest.fn().mockReturnValue(expectedCreatedToken);
+      const ClassToBeInstantiated = delegateServiceUnderTest(undefined);
+
+      Object.setPrototypeOf(ClassToBeInstantiated.prototype, {
+        create: superMethod,
```
Comment (brigonzalez) [inline library/test/setup/utils.ts:652]: 💣

File: library/test/setup/utils.ts [hunk 4]
```diff
@@ -547,6 +547,132 @@ const testCRUD = <T, C, U>(
...
+      });
+      const serviceInstance = new ClassToBeInstantiated({
+        apiKey: chance.string(),
+        baseURL: chance.url(),
+      });
+      const expectedPayload = chance.string(),
+        expectedRequestOptions = chance.string();
+      const createdToken = ((serviceInstance as unknown) as {
+        tokenize?: (...args: unknown[]) => string;
+        create?: (...args: unknown[]) => string;
```
Comment (brigonzalez) [inline library/test/setup/utils.ts:663]: 💣

**Review by brigonzalez — COMMENTED** [review]
💣  away

**Review by lcschy — DISMISSED** [review]

**Review by brigonzalez — DISMISSED** [review]

**Review by lcschy — APPROVED** [review]
🚀

**Pull request merged**